package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "spur-errors",
   "description": "Common error builder utility for Node.js. Contains common error types, and stack trace tracking to support more detailed error messages.",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Is this required to sync a current version out there? Non publish auto bumps up.